packages/types/src/types.d.ts

```diff
@@ -21,7 +21,46 @@ export type OnTransactionHandler = (args: {
 
 export type SnapProvider = MetaMaskInpageProvider;
 
-export type SnapExports = {
+// CAIP2 - https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md
+type ChainId = string;
+// CAIP10 - https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md
+type AccountId = string;
```

Comment:
I think there was a discussion about not using types like this, since they're just wrappers of `string`?

@rekmarks

Reply:
Yeah, there was. This is copied from the SIP, we should decide how we want to specify these.

Reply:
I think given their new type this should be fine.